architecture/src/commonMain/kotlin/observable/Observable.kt

```diff
@@ -381,7 +381,7 @@ interface DefaultObservable<R:T?, T>:BasicObservable<R, T?, Value<R>>, DefaultIn
 
 interface DefaultSubject<R:T?, T>:BasicSubject<R, T?, Value<R>>, DefaultObservable<R, T>, MutableDefaultInitialized<R, T?>
 
-interface WithState<T> {
+expect interface WithState<T> {
```

Comment:
Why it has to be expect? Implementations the same for all platforms. On Android it can be just extended with new liveData property?

Reply:
Extension doesnt work because its not accessible for the databinding view xml.